db/write_thread.h

```diff
@@ -360,6 +360,14 @@ class WriteThread {
   // Enable pipelined write to WAL and memtable.
   const bool enable_pipelined_write_;
 
+  // The lower limit of maximum number of bytes that are written in a single
+  // batch of WAL or memtable write.
+  const uint64_t lower_limit_write_batch_group_size_bytes;
```

Comment:
Consider a better description than "lower_limit". In my understanding, if a write batch is larger than this value, they are not batched? Than maybe something like "max_write_size_to_batch", or something like that?

Reply:
Updated the name to 'max_write_batch_group_size_bytes'